tilequeue/cache/redis_cache_index.py

```diff
@@ -18,7 +18,7 @@ def intersect(self, coords, tiles_of_interest=None):
                 yield coord
 
     def remove_tiles_of_interest(self, coord_ints):
-        return self.redis_client.srem(self.cache_set_key, coord_ints)
+        return self.redis_client.srem(self.cache_set_key, *coord_ints)
```

Comment:
Nice catch! :zap: 

This will turn into a single call to redis, so we probably shouldn't call this with millions of integers. (Not sure if we are or aren't, just caught my eye).